test-wasm/node_shim.js

```diff
@@ -1,12 +1,146 @@
 // This file adds RTCPeerConnection to the global context, making Node.js more
 // closely match the browser API for WebRTC.
 
+const ajv = require("ajv");
```

Comment:
Is there another place where we can add this? __node_shim.js__ is specifically about adding APIs that exist in the browser but not in Node.js. We have not used it so far to pull in third-party libraries outside of that context. For example, we don't load in `BrowserFS` here.

If there is no other place, let's update the comment so that the purpose of this file is clear.